src/Development/IDE/Plugin.hs

```diff
@@ -1,5 +1,9 @@
-
-module Development.IDE.Plugin(Plugin(..), codeActionPlugin, codeActionPluginWithRules) where
+module Development.IDE.Plugin
```

Comment:
Please don’t include unnecessary layout changes in a PR. That just creates noise during review.